logstash-filter-verifier.go

```diff
@@ -50,6 +50,10 @@ var (
 			Flag("logstash-output", "Print the debug output of logstash.").
 			Default("false").
 			Bool()
+	logstashArgs = kingpin.
+			Flag("logstash-arg", "Command line arguments, which are passed to Logstash. Flag and value have to be provided as a flag each, e.g.: --logstash-arg=-n --logstash-arg=InstanceName").
+			Short('l').
```

Comment:
Could we skip the short version of the flag? No other flags have short forms and having it complicates the interface and the documentation (as evidenced by your README.md hunk in this patch)?